### PR TITLE
update to 0.5.0

### PR DESCRIPTION
**Pull request by jessitron**
Removed a bunch of automations that are now in spring-automations.

### CODE REVIEW

File: src/atomist.config.ts [hunk 1]
```diff
@@ -26,15 +21,10 @@ export const configuration: Configuration = {
...
-        VersionSpreadReviewer,
-        VersionMapper,
         NewAutomation,
-        SpringBootModernizer,
-        UpdateCopyright,
-        ReviewCopyright,
+        () => updateCopyrightInOneRepository(),
         // Use a factory if you like...
-        () => new SpringBootVersionUpgrade(),
+        () => new ReviewCopyright(),
```
Comment (ddgenome) [inline src/atomist.config.ts]: Maybe this should be changed to use the auto-scanning?

File: tslint.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
         "max-classes-per-file": false,
         "object-literal-sort-keys": false,
         "no-console": false,
-        "no-var-requires": false
+        "no-var-requires": false,
+        "max-line-length": [true, 150]
```
Comment (ddgenome) [inline tslint.json]: Ugh

**Review by ddgenome — APPROVED** [review]
One optional suggestion.

File: tslint.json (hunk 2 above)
Comment (jessitron) [reply tslint.json]: there's a freaking import that's over 150 characters, with only one class imported

File: src/atomist.config.ts (hunk 1 above)
Comment (jessitron) [reply src/atomist.config.ts]: maybe, but as a separate PR... as a sample I kinda like it being explicit though, it helps _people_ find all the automations

File: tslint.json (hunk 2 above)
Comment (ddgenome) [reply tslint.json:20]: Was it over 120 even when wrapped?

```typescript
import {
    SomeVeryLongExportNameFromAFile,
} from "../../../../../../some/very/long/path/to/an/export/to/import";
```

You can also just disable for that line.

https://palantir.github.io/tslint/usage/rule-flags/

Comment (jessitron) [reply tslint.json:20]: I thought we had a new standard for 150? the 120 length annoys @johnsonr (and me)

**Pull request merged**